tests_e2e/orchestrator/scripts/run-scenarios

```diff
@@ -35,10 +35,15 @@ cp "$HOME/id_rsa" "$HOME/.ssh"
 chmod 700 "$HOME/.ssh/id_rsa"
 ssh-keygen -y -f "$HOME/.ssh/id_rsa" > "$HOME/.ssh/id_rsa.pub"
 
+#
 # Now start the runbook
+#
+lisa_logs="$HOME/logs/lisa"
```

Comment:
Since now we have multiple logs for the full test run, I moved the LISA logs to their own subdirectory. The main LISA log is directly under this subdirectory.